OSS-Task.md

```diff
@@ -0,0 +1,47 @@
+# The Open Source Task
+
+There are a lot of really awesome open source projects out there. If you've ever been a developer, chances are, you have used one. In fact, nearly all the languages themselves are open source!
+
+This task is not just for selections into KOSS. We give you the opportunity to work on any proper open source project, on any major platform (such as GitHub, GitLab, or even e-mail'ed patches) to encourage you to become an OSS contributor. You can even mention this task on your CV if you are a dedicated contributor to any project!
+
+## The Task
+
+You have to create **one Pull Request** on any proper project, Hacktoberfest style!
+
+## Rules
+
+1. The pull request MUST NOT be a trivial one. You cannot just change one line of README or fix indentation.
+2. You can either work on the documentation or the codebase itself. Bear in mind, if you are working on documentation, the result should especially be a significant improvement.
+3. It has to be done on someone else's repository, not your own.
+4. The repository you are working on must atleast be somewhat famous:
+   - It can be a library or application that is not much used, as long as it is actually used by other people.
+   - Big projects are highly welcome.
+   - If you are unsure whether the repository will count towards your selections, feel free to contact us.
+5. Repositories which do not serve any real life purpose (such as [TheAlgorithms](https://github.com/TheAlgorithms) are STRICTLY FORBIDDEN.
+   - This means you cannot just submit a CP-like/algorithmic solution to a project that is just a storage of different algorithms.
+   - You can, however, create a pull request that changes the algorithm of an application or library to make it more efficient.
+6. It is not necessary that the pull requests be merged. Merged requests, however, are highly appreciated.
```

Comment:
Good idea, some projects have a slightly longer cycle to get things merged, but do make sure to test the code in the PR if it's not merged.